inc/impactcompound.class.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+if (!defined('GLPI_ROOT')) {
+   die("Sorry. You can't access this file directly");
+}
+
+/**
+ * @since 9.5.0
+ */
+class ImpactCompound extends CommonDBTM {
```

Comment:
Why an emtpy class?

Reply:
Compounds are used internally to define and store parent relations in the impact graphs but they can't be accessed "directly" in GLPI therefore I don't need to override anything from CommonDBTM.